services/QuillLMS/engines/comprehension/spec/controllers/comprehension/activities_controller_spec.rb

```diff
@@ -70,6 +71,18 @@ module Comprehension
         expect(Activity.count).to(eq(1))
       end
 
+      it "should make a change log record after creating the Activity record" do
+        post :create, params: { activity: { parent_activity_id: activity.parent_activity_id, scored_level: activity.scored_level, target_level: activity.target_level, title: activity.title, notes: activity.notes }}
+
+        new_activity = Activity.last
+        change_log = Comprehension.change_log_class.last
+        assert_equal change_log.serializable_hash["comprehension_action"], "Comprehension Activity - created"
```

Comment:
Since we just converted to `rspec`, we should convert these `minitest` assertions to rspec, e.g. `expect(thing).to_be "this thing"`. Ping @happythenewsad if you'd like guidance here.